src/Router/index.js

```diff
@@ -18,7 +18,18 @@ const router = createRouter({
             path: "/register",
             component: () => import("../views/Register.vue")
 
+        },
+        {
+            path: "/categoryCreate",
```

Comment:
As mentioned the other day the path should be `/category/create`

Reply:
done sir